vulns/aiohttp/PYSEC-2022-43059.yaml

```diff
@@ -1,6 +1,7 @@
 id: PYSEC-2022-43059
 modified: 2023-11-07T20:24:23.321955Z
 published: 2022-06-23T17:15:00Z
+withdrawn: 2023-11-08T00:54:24Z
```

Comment:
Is this the PYSEC date or MITRE date? The NIST website has this in the changelog:
> CVE Modified by MITRE 6/26/2022 4:15:08 PM

Reply:
That was my local date :sweat_smile: -- it might make sense to use the GHSA or CVE date here instead, but IMO it's relatively minor (since they all disagree anyways, and arguably this is the "withdraw date" for *this* advisory DB). But I don't feel strongly about that.

Reply:
For context: the GHSA was withdrawn on June 28 2022, so 2 days after the CVE was last modified.

Reply:
I just wasn't sure if there's guidance / docs on the semantics of these dates.

Reply:
The OSV schema mentions "the entry" meaning PYSEC. Thanks @woodruffw!